modules/media/ajax/FileUpload.php

```diff
@@ -207,6 +207,7 @@ function getUploadFields()
 
     $db   = \NDB_Factory::singleton()->database();
     $user = \User::singleton();
+    $config = \NDB_Config::singleton();
```

Comment:
The equal sign of $config is not aligned with the previous variables. `make checkstatic`